services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -3861,10 +3868,23 @@ protected String metricMessage(String userId,
                                  String operation,
                                  String status,
                                  Long duration,
-                                 String error) {
+                                 String error,
+                                 Double fileSize) {
     StringBuilder res = new StringBuilder("service=webconferencing");
     res.append(" operation=").append(operation);
-    res.append(" status=").append(status);
+    if(operation.equals(OPERATION_CALL_RECORDED) && fileSize != null) {
+      DecimalFormat df = new DecimalFormat("0.00");
+      String fileSizeByMO = df.format(fileSize / 1048576);
+      res.append(", recording_file_size:").append(fileSizeByMO);
+      if (status.equals(STATUS_OK)) {
+        res.append(", upload_successful:").append(true);
+      } else {
+        res.append(", upload_Successful:").append(false);
+      }
+    }
+    else {
+      res.append(" status=").append(status);
+    }
```

Comment:
```suggestion
    res.append(" status=").append(status);
    if(fileSize != null) {
      DecimalFormat df = new DecimalFormat("0.00");
      String fileSizeByMO = df.format(fileSize / 1048576);
      res.append(", recording_file_size:").append(fileSizeByMO);
    }
```

The file size must be moved in parameters part.